### ISSUE TITLE
[3.0.0-Beta3] Archived and Trash View don't appear

### ISSUE THREAD

**Issue by christianhans** [sn-extensions/folders-component#18]
After upgrade (also tried to sign out and sign in again), I can't see the new Archived and Trash views using the folder component. Those two views appear correctly when I deactivate the Folders component. Also, it seems like I have an old "Archived" View that doesn't show any notes.

**Comment by atmoio** [comment]
Correct, Folders hasn't yet been pushed to production. We don't have a beta method of shipping the development version, so you'll see inconsistency with smart tags until the release.
